src/runtime_src/tools/xclbinutil/Section.h

```diff
@@ -34,6 +34,16 @@
 class Section {
  typedef std::function<Section*()> Section_factory;
 
+ public:
+  enum class FormatType{
+    UNDEFINED,
+    UNKNOWN,
+    RAW,
+    JSON,
+    HTML,
+    TXT
+  };
+
  protected:
   class SectionInfo {
```

Comment:
It could be and is does look to be heading towards that direction.  

That said, in a future submission, the collection of these classes will be passed around to helper EoU user routines to describe the sections.  As which time, I wish to have all over the variables to be "read only", something to which they are not now.   

The only reason why they are not behind any getters / setters (which was originaly the case in the previous PR), was because during the code review it was observed that this was more of a data colection.  So we removed them.   

While the structure is a good idea, I am not realy to make it so until the helper routines are completed.   If it turns out at that time it should be one.  I will make it so.